### DISCUSSION TITLE
What's the usage of action?

### DISCUSSION THREAD

**Discussion by Mister-Hope** [ntnyq/vuepress-plugin-social-share#69] [Q&A]
whatsapp has a property `action`, but it seems that the property is used nowhere.

**Comment by ntnyq** [comment]
The networks data is copied from [vue-social-sharing](https://github.com/nicolasbeauvais/vue-social-sharing/blob/588d8182bd3deeaa9edc726d575026e7fe45849f/src/networks.json#L75)

The former whatsapp sharer uses a custom scheme link.

Since we use the https scheme api now, the `action` property can be removed.
